src/application/clients/crux-wallet-client.ts

```diff
@@ -67,43 +85,40 @@ export class CruxWalletClient {
         this.cruxBlockstackInfrastructure = options.blockstackInfrastructure || CruxSpec.blockstack.infrastructure;
         this.walletClientName = options.walletClientName;
         if (options.privateKey) {
-            this._keyManager = typeof options.privateKey === "string" ? new BasicKeyManager(options.privateKey) : options.privateKey;
+            this.keyManager = typeof options.privateKey === "string" ? new BasicKeyManager(options.privateKey) : options.privateKey;
         }
-        this._initPromise = this._init(options);
+        this.initPromise = this.init(options);
     }
 
     @throwCruxClientError
     public getCruxIDState = async (): Promise<ICruxIDState> => {
-        await this._initPromise;
-        if (this._keyManager) {
-            this._cruxUser = await this._cruxUserRepository.getWithKey(this._keyManager, new CruxDomainId(this.walletClientName));
-            if (!this._cruxUser) {
+        await this.initPromise;
+        if (!this.cruxUser) {
+            if (this.keyManager) {
                 return {
                     cruxID: null,
-                    status: {
-                        status: "NONE",
```

Comment:
Why is this being removed?

Reply:
I have added it back. But using the enums.